float32[] pos
float32[] quat

float32 radius
float32 height

